.github/workflows/arm_deb_packager.yml

```diff
@@ -0,0 +1,90 @@
+name: arm_deb_packager
+
+on:
+  push:
+    branches: [main, next]
+    paths:
+      - '**'
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
```

Comment:
I think we changed this slightly in the `next` branch ([here](https://github.com/0xPolygonMiden/miden-node/blob/next/.github/workflows/deb_packager.yml)).

Reply:
Updated  in the arm packager